flang/test/Lower/OpenMP/delayed-privatization-lower-to-llvm.f90

```diff
@@ -0,0 +1,40 @@
+! Tests the OMPIRBuilder can handle multiple privatization regions that contain
+! multiple BBs (for example, for allocatables).
+
+! RUN: %flang -S -emit-llvm -fopenmp -mmlir --openmp-enable-delayed-privatization \
```

Comment:
LLVM IR checks are better avoided in Flang. If we really need this then this should go to the Integration directory. Alternative is to drop it since it will be tested in the Fujitsu suite (later incorporated into llvm-testsuite) and by the translation lit test.